00-dashboard-01

Wallmoreyra opened the pull request:
solo falta implementar la subida de imagen, y estaría completo el form de crear, usando ese modelo, lo adaptaría para el update, falta conectar las rutas

Wallmoreyra committed:
00-dashboard-01

solo falta implementar la subida de imagen, y estaría completo el form de crear, usando ese modelo, lo adaptaría para el update, falta conectar las rutas

peterjbone commented:
ok ahora lo bajo para tratar de implementar lo que sería la subida de imagenes con cloudinary.

Wallmoreyra committed:
00-dashboard-01

Muy lento pero solucione uno de los problemas que mas me estaban dando dolor de cabeza.

Wallmoreyra committed:
00-dashboard-02

Terminado el crud de los productos, a por las verificaciones y los usuarios::
